src/mrpro/operators/LinearOperator.py

```diff
@@ -47,6 +47,53 @@ def H(self) -> LinearOperator:  # noqa: N802
         """Adjoint operator."""
         return AdjointLinearOperator(self)
 
+    def operator_norm(
+        self, initial_value: torch.Tensor, dim: tuple | None = None, max_iterations: int = 64
+    ) -> torch.Tensor:
+        """Power iteration for computing the operator norm of the linear operator.
+
+        Parameters
+        ----------
+        initial_value
+            initial value to start the iteration; if chosen exactly as zero-tensor,
+            we randomly generate an initial value.
+        dim
+            dimensions referring to the dimensions of the tensors on which the operator operates.
+            For example, for a matrix-vector multiplication example,
+            with a batched matrix tensor with shape (4,30,80,160), tensors of shape (4,30,160) to be multiplied,
+            and dim = None, it is understood that the the matrix representation of the considered operator
+            corresponds to a block diagonal operator (with 4*30 matrices) and thus the algorithm returns
+            one single value. In contrast, if for example, dim=(-1), the algorithm computes a batched operator
```

Comment:
a single value or a tensor of shape (1,1,1,1)?

Reply:
actually, of shape (1,1,1)... I have further elaborated the comments.